modular_sand/code/datums/interactions/lewd_definitions.dm

```diff
@@ -802,8 +802,13 @@
 				txt = copytext(A.name, 1, length(A.name))
 			return txt
 
-/// Handles the sex, if cumming returns true.
-/mob/living/proc/handle_post_sex(amount, orifice, mob/living/partner)
+/**
+  * Handles the sex, if cumming returns true.
+  *
+  * Please, make sure to use the correct genital
+  * in the organ argument when you call this proc!
+*/
+/mob/living/proc/handle_post_sex(amount, orifice, mob/living/partner, organ = null)
```

Comment:
this edit might not be necessary since the organ parameter is only used in its modular edit